parsec_web/README.md

```diff
@@ -0,0 +1,356 @@
+# parsec_web [![pub package](https://img.shields.io/pub/v/parsec_web.svg)](https://pub.dev/packages/parsec_web)

```

Comment:
this readme should be simple, keep the complex README only in the main parsec_flutter.

Reply:
Ok! I've tried to make it more concise and focused in the commands.